game/static/game/css/landing.css

```
@@ -289,6 +302,163 @@ body {
+    box-shadow: 0 0 24px rgba(255, 255, 255, 0.85), 0 0 0 6px rgba(240, 192, 64, 0.12);
+    animation: sparklePulse 1s ease-in-out infinite;
+}
+
+.cursor.orbit {
+    width: 12px;
+    height: 12px;
+    background: rgba(240, 192, 64, 1);
+    box-shadow: 0 0 15px rgba(240, 192, 64, 0.4);
+}
+
+@keyframes cursorOrbit {
```

Stylelint (17.13.0)

[error] Expected keyframe name "cursorOrbit" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)